extension/e2e-tests/onboarding.test.ts

```diff
@@ -89,6 +89,57 @@ test("Import wallet", async ({ page }) => {
   await expect(page).toHaveScreenshot("wallet-import-complete-page.png");
 });
 
+test("Import 24 word wallet", async ({ page }) => {
+  await expect(page).toHaveScreenshot("welcome-page.png");
+  await page.getByText("Import Wallet").click();
+  await expect(
+    page.getByText("Import wallet from recovery phrase"),
+  ).toBeVisible();
+  await page.locator(".RecoverAccount__phrase-toggle > label").click();
+
+  // time out to accomodate animation
+  await page.waitForTimeout(150);
```

Comment:
this probably works well enough but wanted to point out that there is also `page.waitForSelector('yourselector')` and `page.waitForLoadState('domcontentloaded')` which could give you less of a racy condition to wait for in the case of waiting for css animations.

Reply:
Ah, nice! That's much better. I was not psyched about doing this, but couldn't get it to work reliably otherwise